lib/src/authentication/widgets/input_password.dart

```diff
@@ -18,14 +19,20 @@ class InputPassword extends StatefulWidget {
 
   final TextInputAction textInputAction;
 
+  final String? Function(String? value) validate;
```

Comment:
To make it optional, no need to enforce it I think 

```suggestion
  final String? Function(String? value)? validate;
```

Reply:
Updated